sys-apps/ignition/ignition-9999.ebuild

```diff
@@ -67,4 +67,8 @@ src_compile() {
 
 src_install() {
 	newbin ${GOBIN}/internal ${PN}
+
+	systemd_dounit "${FILESDIR}/ignition-delete-config.service"
```

Comment:
This file should be part of the regular system, not of the initramfs, or? If it doesn't work to have it added here to the regular system, we could add it in the init repo (otherwise if it's for the initramfs the bootengine repo would fit better).

Reply:
Turns out that it's less trivial than expected - I added the service file to the init repository (https://github.com/flatcar-linux/init/pull/73).

Now comes the fun part:
* `/usr/bin/ignition`: has to be in the initramfs but not in the regular system
* `/usr/libexec/ignition-rmcfg`: has to be in the regular system 

Adding `sys-apps/ignition` to `coreos-base/coreos` dependency will work but we will have `/usr/bin/ignition` in the regular system.

Adding a use-flag like `remove` to have the following dependency `sys-apps/ignition[remove]` won't work because we will have a conflict between `coreos-kernel` and `coreos-base` (since both are installed at the same moment).

Reply:
So, a second package or maybe move this to the initramfs and only use it when a ignition config is used because with cloud-init we wouldn't delete the userdata anyway?

Reply:
I wanted to avoid a second package - since `ignition-rmcfg` _is_ `ignition` but with a different name -> two ebuilds to maintain but I don't see yet other solutions.
Having `ignition-rmcfg` in the initramfs could work but one won't be able to mask the service to prevent the config from being removed.